python/ray/scripts/scripts.py

```diff
@@ -19,6 +21,15 @@
     attach_cluster, exec_cluster, create_or_update_cluster, monitor_cluster,
     rsync, teardown_cluster, get_head_node_ip, kill_node, get_worker_node_ips,
     debug_status, RUN_ENV_TYPES)
+try:
+    from ray.autoscaler._private.get_logs import Archive, \
+        GetParameters, Node, _info_from_params, \
+        create_archive_for_local_and_remote_nodes, \
+        create_archive_for_remote_nodes, get_all_local_data
+    get_logs_available = True
```

Comment:
I just noticed that this introduces a lot of imports into this file. Can you push the logic for `get-logs` and `get-local-logs` into a function in `autoscaler._private.commands`? I think we have tended towards pushing most of the logic in that file